packages/core-ethereum/crates/core-ethereum-db/src/db.rs

```diff
@@ -630,14 +679,15 @@ impl<T: AsyncKVStorage<Key = Box<[u8]>, Value = Box<[u8]>> + Clone> HoprCoreEthe
 
         debug!("start marking {acked_ticket} as redeemed");
 
+        let mut ops = utils_db::db::Batch::default();
+
         let key = get_acknowledged_ticket_key(acked_ticket)?;
-        self.db.remove::<AcknowledgedTicket>(key).await?;
-        debug!("removed {acked_ticket} from the DB");
+        ops.del(key);
 
         let key = utils_db::db::Key::new_from_str(REDEEMED_TICKETS_COUNT)?;
         let count = self.db.get_or_none::<usize>(key.clone()).await?.unwrap_or(0) + 1;
-        self.db.set(key, &count).await?;
-        debug!("updated redeemed tickets count to {count}");
```

Comment:
@NumberFour8 We split these db actions into separate calls because of a perceived issue with batch processing, see comment on top. Is that still a problem?